recipes/cpp-sort/all/conanfile.py

```diff
@@ -40,6 +40,12 @@ def validate(self):
         if self.settings.get_safe("compiler.cppstd"):
             check_min_cppstd(self, self._minimum_cpp_standard)
 
+        if is_msvc(self):
+            if Version(self.version) < "1.10.0":
+                raise ConanInvalidConfiguration("cpp-sort versions older than 1.10.0 do not support MSVC")
+            check_min_vs(self, 192)
```

Comment:
Just an FYI, this internal checks the `is-msvc` so it's not required to be nested :)